GSA_Adapter/CRUD/Create.cs

```diff
@@ -293,6 +294,16 @@ private bool CreateObject(FabricPanelProperty fabricProperty)
         }
 
 
+        /***************************************************/
+        /**** Fallback  Method                         ****/
+        /***************************************************/
+
+        private bool CreateObject(BHoMObject obj)
+        {
+            Compute.RecordError($"Pushing object of type {obj.GetType()} is not implemented for GSA_Toolkit.");
+            return false;
+        }
+
```

Comment:
```suggestion

```

This method should not be added. Conflicts with main method on line 97 of this file as well as method on line 116 for section properties. Please remove.

This is already handled for the general case here:

https://github.com/BHoM/GSA_Toolkit/blob/94f9ece4c98b32834283658dbab59bb122b3dfb8/GSA_Adapter/Convert/ToGsa/_ToGsa.cs#L134-L139